sr/robot3/robot.py

```diff
@@ -203,7 +209,10 @@ def power_board(self) -> PowerBoard:
 
         :return: The power board object
         """
-        return self._power_board
+        if not self._no_pb:
+            return self._power_board
+        else:
+            raise RuntimeError("No power board was initialized")
```

Comment:
Nit: Prefer flatter functions

```suggestion
        if self._no_pb:
            raise RuntimeError("No power board was initialized")

        return self._power_board            
```